### ISSUE TITLE
disable or warn if openmp is enabled

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#2935]
some blas versions use multiple threads. This can be stopped by setting OMP_NUM_THREADS=1, but if you forget to do this, performance is terrible. There is never a reason to have it enabled, I think.

**Comment by bangerth** [comment]
Yes, correct. Didn't we address this somehow in the deal.II test suite?
